### DISCUSSION TITLE
Is there a Foreach method like with gopg/pg?

### DISCUSSION THREAD

**Discussion by adelowo** [uptrace/bun#841] [Q&A]
Hi, I was hoping i could find a `Foreach` method that exists in `go-pg/pg` https://pkg.go.dev/github.com/go-pg/pg/v10#example-DB.Model-ForEach

What is the right way to do that with bun?

**Comment by vmihailenco** [comment]
You can replace ForEach with sql.Rows and db.ScanRow.
